### ISSUE TITLE
AttributeError: module 'tfx.orchestration.kubeflow.kubeflow_dag_runner' has no attribute 'get_default_pod_labels'

### ISSUE THREAD

**Issue by C45513** [tensorflow/tfx#2214]
Hi, I'm running the following code to create a new pipeline in GCP Kubeflow. I'm using the same kubeflow_dag_runner.py file from the taxi demo and ready to create a new pipeline with my own data. Can you please advise? Thanks in advance!

TensorFlow version: 2.2.0
TFX version: 0.21.4
KFP version: 0.5.1

```
!tfx pipeline create  \
--pipeline-path=kubeflow_dag_runner.py \
--endpoint={ENDPOINT} \
--build-target-image={CUSTOM_TFX_IMAGE}
```

However, I've got the following error: 
```
File "kubeflow_dag_runner.py", line 71, in run
    pod_labels = kubeflow_dag_runner.get_default_pod_labels().update(
AttributeError: module 'tfx.orchestration.kubeflow.kubeflow_dag_runner' has no attribute 'get_default_pod_labels'
Error while running "/opt/conda/bin/python3.7 kubeflow_dag_runner.py" 
```

**kubeflow_dag_runner.py**
```
runner_config = kubeflow_dag_runner.KubeflowDagRunnerConfig(
      kubeflow_metadata_config=metadata_config, tfx_image=tfx_image)
  pod_labels = kubeflow_dag_runner.get_default_pod_labels().update(
      {telemetry_utils.LABEL_KFP_SDK_ENV: 'tfx-template'})
  kubeflow_dag_runner.KubeflowDagRunner(
      config=runner_config, pod_labels_to_attach=pod_labels
  ).run(
      pipeline.create_pipeline(
          pipeline_name=configs.PIPELINE_NAME,
          pipeline_root=PIPELINE_ROOT,
          data_path=DATA_PATH,
          # TODO(step 7): (Optional) Uncomment below to use BigQueryExampleGen.
          # query=configs.BIG_QUERY_QUERY,
          preprocessing_fn=configs.PREPROCESSING_FN,
          run_fn=configs.RUN_FN,
          train_args=trainer_pb2.TrainArgs(num_steps=configs.TRAIN_NUM_STEPS),
          eval_args=trainer_pb2.EvalArgs(num_steps=configs.EVAL_NUM_STEPS),
          eval_accuracy_threshold=configs.EVAL_ACCURACY_THRESHOLD,
          serving_model_dir=SERVING_MODEL_DIR,
          # TODO(step 7): (Optional) Uncomment below to use provide GCP related
          #               config for BigQuery with Beam DirectRunner.
          # beam_pipeline_args=configs
          # .BIG_QUERY_WITH_DIRECT_RUNNER_BEAM_PIPELINE_ARGS,
          # TODO(step 8): (Optional) Uncomment below to use Dataflow.
          # beam_pipeline_args=configs.DATAFLOW_BEAM_PIPELINE_ARGS,
          # TODO(step 9): (Optional) Uncomment below to use Cloud AI Platform.
          # ai_platform_training_args=configs.GCP_AI_PLATFORM_TRAINING_ARGS,
          # TODO(step 9): (Optional) Uncomment below to use Cloud AI Platform.
          # ai_platform_serving_args=configs.GCP_AI_PLATFORM_SERVING_ARGS,
      ))
```

**Comment by jiyongjung0** [comment]
Thank you for the report, and sorry for your inconvenience. This bug is fixed in TFX 0.22.0 or later.
Or you might be able to workaround the problem by deleting `pod_labels` and `pod_labels_to_attach`.

**Comment by rmothukuru** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
